### PR TITLE
Switch 3b1b Linear Algebra from pre to corequisite

### PR DESCRIPTION
**Pull request by Alaharon123**
Switch 3b1b Essence of Linear Algebra from prerequisite of MIT Linear Algebra to corequisite

### CODE REVIEW

**Comment by Alaharon123** [comment]
This has been mentioned a couple times in the discord, how it's better to go through these simultaneously than to watch all of Essence of Linear Algebra before starting MIT Linear Algebra. I can make an RFC if you want, I just figure the change is small enough that it's not necessarily needed

**Comment by waciumawanjohi** [comment]
I'm amenable to the change. Should we remove the separate line for essence of linear algebra and make the corequisite a link to the series?

**Comment by Alaharon123** [comment]
How do you propose doing that?

**Comment by oNjEncUREn** [comment]
Personally, I'm good with the way the commit presents the co-requisite; however, going forward, should this be the driver for adding another column in the table?  Maybe not if this is just a one-off sort of thing.

**Comment by bradleygrant** [comment]
I don't know that the 3B1B material is a /*requisite as much as it's *supplementary material* to the Linear Algebra course.

- Gilbert Strang's 18.06 course is complete without the 3B1B material
- the 3B1B material is not complete on its own
- we've recommended the 3B1B material be consumed alongside/during the 18.06 course to provide additional intuition
- the 3B1B material is undoubtedly useful and helpful, and I (and others) highly recommend people give it a look
- but ultimately, it's not *needed* to develop a usable intuition in linear algebra.

Bottom line: I wouldn't call it a (pre, co) requisite at all. It's best categorized as highly-recommended supplementary material.

Some of Josh Starmer's StatQuest! videos, especially his treatment of PCA, could just as easily be recommended alongside (or instead of) the 3B1B series.
